.github/workflows/release.yaml

```diff
@@ -0,0 +1,194 @@
+        run: make ${{ matrix.command }}
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.directories }}
+          path: shim/bin/
+        
+  kuasar-image:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build kuasar.img
+        run: RUNTIME=docker make bin/kuasar.img
+        working-directory: .
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: kuasar.img
+          path: bin/
+  
+  vmlinux-bin:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build vmlinux.bin
+        run: make bin/vmlinux.bin
+        working-directory: .
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: vmlinux.bin
+          path: bin/
+
+  containerd:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    env:
+      CONTAINERD_VERSION: "1.7.0"
+      CONFIG_FILE: "config.toml"
+      BIN_DIR: "bin"
+    steps:
+      - name: Download and Extract Containerd
+        run: |
+          curl -LJO https://github.com/containerd/containerd/releases/download/v${{ env.CONTAINERD_VERSION }}/containerd-${{ env.CONTAINERD_VERSION }}-linux-amd64.tar.gz
+          mkdir ${{ env.BIN_DIR }} && tar -C ${{ env.BIN_DIR }} -xzvf containerd-${{ env.CONTAINERD_VERSION }}-linux-amd64.tar.gz
+      - name: Generate Config File
+        run: |
+          cat << EOL | sudo tee ${{ env.BIN_DIR }}/${{ env.CONFIG_FILE }}
+          version = 2
+
+          [plugins."io.containerd.grpc.v1.cri"]
+            disable_apparmor = true
+
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.vmm]
+            runtime_type = "io.containerd.kuasar.v1"
+            sandboxer = "vmm"
+            io_type = "hvsock"
+
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.quark]
+            runtime_type = "io.containerd.quark.v1"
+            sandboxer = "quark"
+
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.wasm]
+            runtime_type = "io.containerd.wasm.v1"
+            sandboxer = "wasm"
+
+          [proxy_plugins.vmm]
+            type = "sandbox"
+            address = "/run/vmm-sandboxer.sock"
+
+          [proxy_plugins.quark]
+            type = "sandbox"
+            address = "/run/quark-sandboxer.sock"
+
+          [proxy_plugins.wasm]
+            type = "sandbox"
+            address = "/run/wasm-sandboxer.sock"
+          EOL
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: containerd
+          path: ${{ env.BIN_DIR }}
+
+  release:
+    permissions:
+      contents: write
+    needs:
+      - buildSandboxer
+      - buildShim
+      - kuasar-image
+      - vmlinux-bin
```

Comment:
We have `vmm quark wasm` target in Makefile. Maybe `make vmm`, `make quark` and `make wasm` could make it simple.